app/config/config.yml

```diff
@@ -96,25 +96,34 @@ smartbox_core:
 smartbox_integration_framework:
     defer_events_to_uri: "queue://events"
     queue_drivers:
-        rabbitmq:
-            type: RabbitMQ
-            description: RabbitMQ queue driver
+#        rabbitmq:
```

Comment:
Do we need dead code "The Sixth Sense" style?

Reply:
ha ha.. it is there for demonstration purposes, as with most of the "extras" in the Skeleton Bundle. Just so it is easy for others to see the possibilities.